apps/site/src/app/(main)/(home)/sections/FAQ/FAQAccordionDesktop.tsx

```diff
@@ -0,0 +1,140 @@
+"use client";
+
+import React, { useState } from "react";
+import Image from "next/image";
+
+import ListItemButton from "./components/ListItemButton";
+import TriangleIcon from "./components/TriangleIcon";
+import { FAQAccordion, FAQ } from "./FAQ";
+
+import SpeechDesktop from "./assets/speech-desktop.svg";
+
+const FAQAccordionDesktop: React.FC<FAQAccordion> = ({ faq }) => {
+	const faqGroup1 = faq.slice(0, 8);
+	const faqGroup2 = faq.slice(8);
+
+	const [page1Selected, setPage1Selected] = useState<boolean>(true);
+
+	const [focusedQuestion, setFocusedQuestion] = useState<null | FAQ>(null);
+
+	return (
+		<div className="flex flex-1 justify-center lg:h-auto mt-16 lg:mt-8 mb-[-250px] lg:mb-0">
+			{/* Desktop View */}
+			<Image
+				src={SpeechDesktop}
+				alt="Dialogue box background"
+				className="hidden lg:block h-fit  lg:max-w-[630px] lg:h-[310px] xl:max-w-[775px] xl:h-[350px] duration-300"
+			/>
+			<div className="hidden lg:block absolute lg:w-[555px] xl:w-[685px] lg:text-[.85rem] xl:text-[1rem] mt-3">
+				{/* Page 1 focused */}
+				<div
+					className={`absolute w-full duration-300 ${
+						page1Selected && !focusedQuestion
+							? "opacity-100 z-50"
+							: "opacity-0 pointer-events-none"
```

Comment:
Suggestion [repeating]: rather than using `opacity` to show and hide stuff, use `hidden` instead